src/main/java/bio/terra/cli/service/DataRepoService.java

```diff
@@ -38,7 +39,9 @@ private DataRepoService(@Nullable User user, Server server) {
     if (user != null) {
       // fetch the user access token
       // this method call will attempt to refresh the token if it's already expired
-      this.apiClient.setAccessToken(user.getUserAccessToken().getTokenValue());
+      AccessToken token =
+          server.getIdTokenAuthentication() ? user.getUserIdToken() : user.getUserAccessToken();
```

Comment:
You have to do this ternary bit twice; maybe make a method like `getEffectiveToken()`.

Reply:
I still like `user.getToken` (or `user.getEffectiveToken`). The fact that `user.getToken` uses Server is an implementation detail. (One that will hopefully go away in the future.) Conceptually, you're getting a token for the `user` that was passed in.